feat: added multichain support for rainbow decoder (#9647)
rotki/rotki: docs/changelog.rst

Changelog
=========

> * :feature:`9600` rotki will now correctly decode swap going through rainbow router on Mainnet, Arbitrum one, Base, Binance Smart Chain, Optimism and Polygon PoS.

Reviewer: ```suggestion
* :feature:`9600` rotki will now correctly decode swaps going through the rainbow router on all supported chains.
```
